tsl/src/nodes/gapfill/exec.c

```diff
@@ -23,18 +24,12 @@
 #include <utils/syscache.h>
 #include <utils/typcache.h>
 
-#include "compat.h"
-#if PG12_LT
-#include <optimizer/var.h>
-#else
-#include <optimizer/optimizer.h>
-#endif
-
 #include "nodes/gapfill/gapfill.h"
 #include "nodes/gapfill/locf.h"
 #include "nodes/gapfill/interpolate.h"
 #include "nodes/gapfill/exec.h"
 #include "time_bucket.h"
+#include "compat.h"
```

Comment:
Not sure why you're keeping it here: you've removed it everywhere else. What function is it needed for?

Reply:
needed for TTSOpsVirtualP

Reply:
Ok.